res/values/strings.xml

```diff
@@ -82,6 +82,8 @@
     <string name="file_list_file">file</string>
     <string name="file_list_files">files</string>
     <string name="filedetails_select_file">Tap on a file to display additional information.</string>
+    <string name="upload_owncloud">Upload to ownCloud</string>
```

Comment:
the word "ownCloud" should be parameterized with the value in `setup.xml` file "app_name"